.travis.yml

```diff
@@ -63,7 +63,7 @@ before_script:
   - cmake .. -DCMAKE_INSTALL_PREFIX=/usr;
   - make -j 3
   - # AppImage generation
-  - if [ "$TRAVIS_BRANCH" = "master" ] && [ "$CC" = "clang-4.0" ] && [ "$TRAVIS_PULL_REQUEST" = false ]; then
+  - if [ "$TRAVIS_REPO_SLUG" = "RPCS3/rpcs3" ] && [ "$TRAVIS_BRANCH" = "master" ] && [ "$CC" = "clang-4.0" ] && [ "$TRAVIS_PULL_REQUEST" = false ]; then
```

Comment:
could you instead change this to `[ -n "$UPLOAD_URL" ]`, because I have UPLOAD_URL set for my travis ci to point to https://transfer.sh so that I can provide AppImages for linux users to test.

Reply:
But what about other forks?
I keep getting spam from travis about build failure since it cannot upload from my fork